velox/type/tests/TimestampConversionTest.cpp

```diff
@@ -104,6 +104,100 @@ TEST(DateTimeUtilTest, fromDateInvalid) {
       1970, 6, 31, "Date out of range: 1970-6-31"));
 }
 
+TEST(DateTimeUtilTest, fromWeekOfMonthDate) {
+  auto daysSinceEpochLenient =
```

Comment:
nit: daysSinceEpochLenient -> daysSinceEpoch for readability

fromWeekOfMonthDate -> daysSinceEpochFromWeekOfMonthDateLenient

Reply:
Thanks. Updated.